scripts/package.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
```

Comment:
Just to align on the same understanding - you don't expect this to script to be run in the tests or packaging pipeline (yet)?

Reply:
I'm working on some refactoring to the `integration.sh` script to include the use of the `packaging.sh`. Do you have in mind any problem that may occur in the IC? locally it works perfectly for the moment.

Reply:
(That's part of a different task, not related to the one in this PR)

Reply:
SGTM.

> Do you have in mind any problem that may occur in the CI?
No, just make sure to align the packaging part https://github.com/cloudfoundry/buildpacks-ci/blob/master/tasks/detect-and-upload/buildpack-tagger.rb#L39